### PR TITLE
Flash message fixes

### LINKED ISSUE #1526: Make Tor Browser logout instructions more user friendly
This is a UX suggestion by @ninavizz to improve the notification displayed to sources upon logout (click on image below to see her suggestion for this issue in greater detail (and a nice icon we can use)):

screen shot 2017-01-17 at 5 08 18 pm

### PR DESCRIPTION
**Pull request by heartsucker**
Fixes #1526, #1525, and #1524.

Doesn't include adding the custom org name to one of the flashed messages.

Some of this was pending until after #1538 got merged. Y'know, not to pile too much into one PR.

### CODE REVIEW

**Comment by heartsucker** [comment]
Also @ninavizz is going to want to check these for colors since I guessed for some since values weren't (aren't?) in the relevant issues.

File: securedrop/source.py [hunk 1]
```diff
@@ -292,18 +293,24 @@ def submit():
                 fh.stream))
 
     if first_submission:
-        flash(
-            "Thanks for submitting something to SecureDrop! Please check back later for replies.",
-            "notification")
+        flash(Markup("""{svg}<div class="message"><strong>Success!</strong>
+                     <p>Thank you for sourcing this information to us.
+                     Please check-back later for replies. <a href="#codename-hint">
```
Comment (redshiftzero) [inline securedrop/source.py]: 'check-back' -> 'check back'

File: securedrop/source.py [hunk 2]
```diff
@@ -292,18 +293,24 @@ def submit():
                 fh.stream))
 
     if first_submission:
-        flash(
-            "Thanks for submitting something to SecureDrop! Please check back later for replies.",
-            "notification")
+        flash(Markup("""{svg}<div class="message"><strong>Success!</strong>
+                     <p>Thank you for sourcing this information to us.
```
Comment (redshiftzero) [inline securedrop/source.py]: 'sourcing' -> 'sending'

File: securedrop/source.py [hunk 3]
```diff
@@ -392,17 +399,14 @@ def login():
...
-                           <img src='static/i/toronion.png' alt='Tor icon' />
-                           Tor onion icon in the toolbar above.<br>
-                         2. Click <strong>  New Identity</strong>.<br>
-                         3. Click <strong>Yes</strong> in the dialog box
-                           that appears.""")
-        flash(tor_msg, "error")
+        msg = Markup("""<div class="icon">{svg}</div>
+                     <div class="message"><strong>Important!</strong><br>
+                     <p>Thank you for exiting your session! Please select "New
+                     Identity" from the green Onion button in the Tor browser,
```
Comment (redshiftzero) [inline securedrop/source.py]: delete comma at end of line

File: securedrop/source_templates/lookup.html [hunk 4]
```diff
@@ -5,7 +5,19 @@
...
 
   {% if flagged and not haskey %}
-  <p class="notification"> <img class="pull-left" src="{{ url_for('static', filename='i/font-awesome/black/info-circle.svg') }}" width="20px" height="20px">A journalist has been waiting for you to log in again so SecureDrop can generate a crypto key for you. Now that you have logged in, they are able to write you a reply. Check back later for replies.</p>
+    <div class="flash notification">
+      <img src="{{ url_for('static', filename='i/relieved_face.svg')  }}" alt="relieved-face" class="icon">
+      <div class="message">
+        <strong>Whew, it’s you! Now, the embarrassing part...</strong></br>
+        <p>Our servers experienced an unusual surge of new activity, when you last visited. This could
+          have been human activity, an automated attack, or just some random blip. To err on the side
+          of caution, we put a hold on sending all documents from that day, through to our journalists.</p>
```
Comment (redshiftzero) [inline securedrop/source_templates/lookup.html]: 'day, through' -> 'day through'

File: securedrop/source_templates/lookup.html [hunk 5]
```diff
@@ -5,7 +5,19 @@
...
+
+        <p>Now that we know you’re really a human, though, we’ll get your previous submission into the
+          hands of a journalist, straight away. We’re sorry for the delay. Please do check-back again
```
Comment (redshiftzero) [inline securedrop/source_templates/lookup.html]: 'journalist, straight away' -> 'journalist straight away'

Comment (redshiftzero) [inline securedrop/source_templates/lookup.html]: 'check-back' -> 'check back'

File: securedrop/tests/test_unit_util.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+    def test_svg_bad_extension(self):
+        with source.app.app_context():
+            with self.assertRaises(PathException):
+                util.svg('config.py')
+
+    def test_svg_bad_path(self):
+        with source.app.app_context():
+            with self.assertRaises(PathException):
+                util.svg('../../../../etc/hosts')
```
Comment (redshiftzero) [inline securedrop/tests/test_unit_util.py]: :+1:

File: securedrop/sass/_base.sass [hunk 7]
```diff
@@ -322,6 +334,15 @@
...
         color: #D62727
 
+    &.important
+      border: 1px solid #EBDCEB
+      background: #FDFAFD
+
+      strong
+        color: #673466
+      p
+        color: #555555
```
Comment (redshiftzero) [inline securedrop/sass/_base.sass]: It's probably best to store these hex color codes as variables in  `_variables.sass`

Comment (heartsucker) [inline securedrop/sass/_base.sass]: If we add these colors anywhere else, I'd do it, I feel like for the moment these are specific enough to the single place they're used to leave them here. Also, according to @ninavizz 's mock ups, we need to have text and link colors change which is actually going to require a chunk of alterations to how we structure the html/css if we want it to make sense.

**Comment by redshiftzero** [comment]
Ok, changes look good now. Merging...

**Comment by ninavizz** [comment]
Thanks for cc'ing me @heartsucker — still getting my chops sharpened with this whole Terminal/Git/Unix/etc stuff, to check PRs... so, it'll likely not be until mid-week at the earliest, that I'll be able to look at PRs on my own. Until then—to meltdowns and the ultra-appreciated encouragement and help from @fowlslegs! :)

**Pull request merged**